com.github.textosaurus.json

```diff
@@ -0,0 +1,46 @@
+{
+    "build-options": {
+        "env": {
+            "CC": "/usr/lib/sdk/gcc7/bin/gcc",
+            "CXX": "/usr/lib/sdk/gcc7/bin/g++"
+        }
+    },
+    "command": "textosaurus",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host"
+    ],
+    "app-id": "com.github.textosaurus",
+    "modules": [
+        {
+            "build-commands": [
+                "mkdir -p /app",
+                "$CXX --version",
+                "qmake -v",
+                "qmake textosaurus.pro QMAKE_CXX=$CXX -r CONFIG+=release PREFIX=/app",
+                "make -j3",
+                "make install"
+            ],
+            "buildsystem": "simple",
```

Comment:
As I said before, `simple` isn't needed. Use `qmake` and set `config-opts`.

Reply:
Is there somewhere docs for qmake buildsystem?

Reply:
Not specifically. Here is an example:

```json
  {
    "buildsystem": "qmake",
    "config-opts": [
      "CONFIG+=release"
    ]
   }
```